engines/gob/detection/tables_gob1.h

```diff
@@ -602,6 +602,44 @@
 	0, 0, 0
 },
 
+// -- Amiga --
+
+{
+	{
+		"gob1",
+		"",
+		AD_ENTRY5s("intro.stk", "c65e9cc8ba23a38456242e1f2b1caad4", 135561,
+		           "disk1.stk", "6797b5dbdace1518ecf93dc0414970d8", 367110,
+		           "disk2.stk", "974e3c12f652adcaf3fba1a87a8cc429", 336641,
+		           "disk3.stk", "ce558c9edd76614c4fe4bedcdb42aba2", 317232,
+		           "disk4.stk", "be08d48dd6307e3a4a0cda97920149d1", 362638),
+		UNK_LANG,
+		kPlatformAmiga,
+		ADGF_NO_FLAGS,
+		GUIO2(GUIO_NOSUBTITLES, GUIO_NOSPEECH)
+	},
+	kFeaturesNone,
+	0, 0, 0
+},
+{
+	{
+		"gob1",
+		"",
+		AD_ENTRY5s("intro.stk", "c65e9cc8ba23a38456242e1f2b1caad4", 135561,
+		           "disk1.stk", "a6ed3c1c9a46c511952bac0c11c691f5", 367048,
```

Comment:
this is correct but the disk2-4 files are redundant. You can stop at disk1 and use AD_ENTRY2s

Reply:
its now been fixed